api/src/test/java/com/epam/pipeline/acl/dts/DtsOperationsApiServiceTest.java

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
It is not directly connected with the pull request but it would be nice to refactor a bit. Could you please also replace `hasRole('ADMIN') OR hasRole('ROLE_USER')` with `AclExpressions.ADMIN_OR_GENERAL_USER` in `DtsOperationsApiService` similarly to `DtsRegistryApiService`.